preferencesfx/src/main/java/com/dlsc/preferencesfx/model/PreferencesFxModel.java

```diff
@@ -89,22 +89,20 @@ public PreferencesFxModel(
    * translated properly according to the TranslationService used.
    */
   private void initializeCategoryTranslation() {
-    flatCategoriesLst.forEach(category -> {
-      translationServiceProperty().addListener((observable, oldValue, newValue) -> {
-        category.translate(newValue);
-        // listen for i18n changes in the TranslationService for this Category
-        newValue.addListener(() -> category.translate(newValue));
-      });
-    });
+    categoriesFlat.forEach(category -> translationServiceProperty().addListener((observable, oldValue, newValue) -> {
+	  category.translate(newValue);
+	  // listen for i18n changes in the TranslationService for this Category
+	  newValue.addListener(() -> category.translate(newValue));
+	}));
   }
 
   private void createBreadcrumbs(List<Category> categories) {
     categories.forEach(category -> {
-      if (!Objects.equals(category.getGroups(), null)) {
+      if (category.getGroups() != null) {
```

Comment:
what about `Objects.isNull(...)`?

Reply:
Would be an option, but I don't see a need to call a method for such a simple operation ^^

Reply:
Makes sense :wink: